### PR TITLE
Byte storage api support

### PR DESCRIPTION
**Pull request by generall**
Implement API integration of byte storage

### CODE REVIEW

**Review by IvanPleshkov — APPROVED** [review]
I like `VectorParamsBuilder`, it makes tests better. I left some nitpicks but in general no concerns

File: lib/collection/src/operations/conversions.rs [hunk 1]
```diff
@@ -1610,6 +1631,18 @@ impl From<VectorParams> for api::grpc::qdrant::VectorParams {
             hnsw_config: value.hnsw_config.map(Into::into),
             quantization_config: value.quantization_config.map(Into::into),
             on_disk: value.on_disk,
+            datatype: value
+                .datatype
+                .map(|dt| api::grpc::qdrant::Datatype::from(dt).into()),
```
[the remark this answers is not in the document]
Comment (generall) [reply lib/collection/src/operations/conversions.rs:1636]: yes

**Comment by raulcarlomagno** [comment]
does it support float16? or just float32 and int8?

**Comment by generall** [comment]
just float32 and int8 for now

**Pull request merged**